trashtalk/views/cleanups.py

```diff
@@ -5,9 +5,10 @@
 
 from geopy.exc import GeopyError
 
+from trashtalk.google_sheets import send_to_sheet
 from trashtalk.seeclickfix import postSCFix
 from trashtalk.factories import cleanup_factory, location_factory
-from trashtalk.models import Cleanup, db_session  # User
+from trashtalk.models import Cleanup, db_session, User
```

Comment:
`User` is not being used. We can remove this import.

Reply:
Sure, but there was like a good *5 min* where it seemed pretty vital. (It's gone now)